server_integration_test.go

```diff
@@ -567,6 +604,41 @@ func TestServerGet(t *testing.T) {
 	}
 }
 
+func TestServerRelativeGet(t *testing.T) {
+	tmpWorkingDir, err := ioutil.TempDir("", "sftp")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(tmpWorkingDir)
+
+	listenerGo, hostGo, portGo := testServer(t, GOLANG_SFTP, READONLY, tmpWorkingDir)
+	defer listenerGo.Close()
+
+	fn := randName()
+	tmpFileLocal := filepath.Join("/tmp", fn)
+	defer os.RemoveAll(tmpFileLocal)
+
+	t.Logf("get: local %v remote %v ", tmpFileLocal, fn)
+
+	// create a file with random contents. This will be the remote file pulled from the server
+	tmpFileRemoteData := randData(10 * 1024 * 1024)
```

Comment:
I don't see what this is testing ? Why do we need to write 10mb of zeros, sftp it from the sever to the same tmpdir locally. Isn't this about testing path names ?